lib/connection/connection.d.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2015-2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+export enum QueryStatus {
```

Comment:
should the enum be declared in the module?

Reply:
Initially, I declared enums in the module. In that case, however, the user can see the enum lists in the snowflake module. In my understanding, the enum lists should be shown when they search the function list in IDE. WDYT?